cogeo_mosaic/backends/dynamodb.py

```diff
@@ -48,6 +48,12 @@ def point(self, lng: float, lat: float) -> List[str]:
         tile = mercantile.tile(lng, lat, self.quadkey_zoom)
         return self.get_assets(tile.x, tile.y, tile.z)
 
+    @property
+    def quadkeys(self) -> List[str]:
+        """Return the list of quadkey tiles."""
+        resp = self.table.scan(ProjectionExpression="quadkey")
```

Comment:
Not sure how costly are the `scan` operation! maybe we could add a warning here ?

Reply:
Yeah I would suggest a warning.

I know that it's standard to return a `list`, but I'd suggest seeing how easy it is to return an iterator here.